packages/eslint-plugin-udemy/rules/no-settimeout/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+module.exports.rules = {
+    'no-settimeout': {
+        create(context) {
+            return {
+                ExpressionStatement(node) {
+                    if (node.expression.callee && node.expression.callee.name === 'setTimeout') {
+                        context.report({
+                            node,
+                            message: 'Found `setTimeout` used in a spec. Please use `await delay` instead.',
```

Comment:
This says "spec", but there's nothing about the rule that is specific to specs, right?

Reply:
Should there be parentheses after `delay()`? Otherwise, the code will actually work, but it'll be subtly broken.

Reply:
Okay, I fixed the wording of the error report.